cmd/interrupt_windows.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"os"
+	"os/exec"
+	"sync"
+	"syscall"
+)
+
+// interruptProcess sends a CTRL-BREAK to the process.
+func interruptProcess(proc *os.Process) error {
+	return sendCtrlBreak(proc.Pid)
+}
+
+var (
+	kernel32      *syscall.DLL
+	ctrlBreakProc *syscall.Proc
+	kernelOnce    sync.Once
+	kernelErr     error
+)
+
+func sendCtrlBreak(pid int) error {
+	kernelOnce.Do(func() {
+		if kernel32, kernelErr = syscall.LoadDLL("kernel32.dll"); kernelErr != nil {
+			return
+		}
+		ctrlBreakProc, kernelErr = kernel32.FindProc("GenerateConsoleCtrlEvent")
+	})
+	if kernelErr != nil {
+		return kernelErr
+	}
+
+	r, _, err := ctrlBreakProc.Call(syscall.CTRL_BREAK_EVENT, uintptr(pid))
+	if r == 0 {
+		return err
+	}
+
+	return nil
+}
+
+func suppressCtrlBreak(err error) error {
+	if exitErr, ok := err.(*exec.ExitError); ok {
+		if exitErr.ExitCode() == 3221225786 {
```

Comment:
@pkedy What is this number?

Reply:
I'm adding a constant for this but that is the exit/status code for Ctrl-Break.

Reply:
Also please add any documentation links for references as comments.